appveyor.yml

```diff
@@ -15,9 +15,15 @@ install:
   - rustc -vV
   - cargo -vV
 build_script:
-  - cargo build --features vulkan
+  - >
+    cargo build --all --features vulkan
+    --exclude gfx_window_glfw
+    --exclude gfx_device_metal
+    --exclude gfx_device_metalll
+    --exclude gfx_window_metal
 test_script:
-  - cargo test --all --features vulkan
-  - cargo test -p gfx -p gfx_core --features "mint serialize"
-  - if %COMPILER%==msvc cargo test --all --features dx11
-  - if %COMPILER%==msvc cargo test --all --features dx12
+  # re: gfx is gfx_render
+  - cargo test -p gfx -p gfx_core
```

Comment:
Appveyor should transit to `cargo test --all --excludes ...` later as not everything is tested as far as I can see. But that requires some more changes to the current `gfx_window_dxgi`, which we can/should address later.